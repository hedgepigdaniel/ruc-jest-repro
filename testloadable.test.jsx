// setup file
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';

configure({ adapter: new Adapter() });


import React from 'react';
import { shallow } from 'enzyme';

import TestLoadableComponent from './testloadable.jsx';


describe('<TestLoadableComponent />', () => {
  it('renders', () => {
    const r = shallow(
      <TestLoadableComponent />
    );
    expect({
      nodes: r.getElements(),
      state: r.state(),
    }).toMatchSnapshot();
  });
});
