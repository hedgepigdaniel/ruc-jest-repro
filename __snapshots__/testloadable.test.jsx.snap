// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TestLoadableComponent /> renders 1`] = `
Object {
  "nodes": Array [
    <DefaultLoading />,
  ],
  "state": Object {
    "error": null,
  },
}
`;
